bindings/python/tests/fdbcli_tests.py

```diff
@@ -216,6 +217,9 @@ def kill(logger):
 
 @enable_logging()
 def suspend(logger):
+    if not shutil.which("pidof"):
+        logger.debug("Skipping suspend test. Pidof not available")
+        return
```

Comment:
Is this the case in our CI, such that we will stop running the test there?

Reply:
I remember we have it in the CI, not sure if it's changed.
But it's not available on macOS and makes the test fail on macOS.
Add this may make the test work on macOS I guess?

Reply:
This is the case in joshua agents, but in CI. We could probably add `pidof` to the Joshua agents. I'll follow up on that